src/Jackett.Common/Indexers/ShizaProject.cs

```diff
@@ -193,7 +195,13 @@ private async Task<List<ReleaseInfo>> FetchShowReleases(string url)
         private bool IsAuthorized(WebResult result) {
             var parser = new HtmlParser();
             var document = parser.ParseDocument(result.ContentString);
-            return document.QuerySelector("div.profile-menu > a").Attributes["href"].Value.EndsWith("/logout");
+            try {
+                return document.QuerySelector("div.profile-menu > a").Attributes["href"].Value.EndsWith("/logout");
```

Comment:
IMHO you can simplify the code. Just string.Contains is enough and less error prone.
https://github.com/Jackett/Jackett/blob/2ea22af9efeb2ec85c2cfb93c9882f8a616843fb/src/Jackett.Common/Indexers/Nebulance.cs#L102